<link href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" rel="stylesheet">

#### Welcome to the CKD-Application
This app is ...

<i class="far fa-comment"></i> designed to interactively explore the results of <a href="https://www.biorxiv.org/content/early/2018/02/14/265447" target="_blank">*A functional landscape of chronic kidney disease entities from public transcriptomic data*</a>.

<i class="fas fa-user-cog"></i> developed and maintained by Christian Holland (<a href="mailto:christian.holland@bioquant.uni-heidelberg.de" target="_blank"><i class="glyphicon glyphicon-envelope"></i></a>) at the <a href="http://saezlab.org" target="_blank">Saezlab</a>, Institute for Computational Biomedicine, University of Heidelberg.

<i class="fas fa-laptop-code"></i> developed using <a href="https://shiny.rstudio.com/Shiny" target="_blank">Shiny</a>.

---

<i class="fab fa-github"></i> Source code is available on <a href="https://github.com/saezlab/ShinyCKD" target="_blank">GitHub</a>.

<i class="fas fa-question"></i> Please use the <a href="https://github.com/saezlab/ShinyCKD/issues" target="_blank">GitHub's issue system</a> to address questions, bugs or feedback. 

---

<center>
<a href="http://saezlab.org" target="_blank"><img src="https://github.com/saezlab/ShinyCKD/blob/master/www/logo_saezlab.png" width="20%", align="center"></a>

<i class="fab fa-twitter"></i> <a href="https://twitter.com/sysbiomed?lang=de" target="_blank">@sysbiomed</a>
</center>

---

#### A functional landscape of chronic kidney disease entities from public transcriptomic data
**Abstract:** To develop efficient therapies and identify novel early biomarkers for chronic kidney disease an understanding of the molecular mechanisms orchestrating it is essential. We here set out to understand how differences in CKD origin are reflected in gene expression. To this end, we integrated publicly available human glomerular microarray gene expression data for nine kidney disease entities that account for a majority of CKD worldwide. We included data from five distinct studies and compared glomerular gene expression profiles to that of non-tumor parts of kidney cancer nephrectomy tissues. A major challenge was the integration of the data from different sources, platforms and conditions, that we mitigated with a bespoke stringent procedure. This allowed us to perform a global transcriptome-based delineation of different kidney disease entities, obtaining a landscape of their similarities and differences based on the genes that acquire a consistent differential expression between each kidney disease entity and nephrectomy tissue. Furthermore, we derived functional insights by inferring activity of signaling pathways and transcription factors from the collected gene expression data, and identified potential drug candidates based on expression signature matching. We validated representative findings by immunostaining in human kidney biopsies indicating e.g. that the transcription factor FOXM1 is significantly and specifically expressed in pariethal epithelial cells in RPGN whereas not expressed in control kidney tissue. These results provide a foundation to comprehend the specific molecular mechanisms underlying different kidney disease entities, that can pave the way to identify biomarkers and potential therapeutic targets. To facilitate this, we provide our results as a free interactive web application.

**Workflow**:
<img src="https://github.com/saezlab/ShinyCKD/blob/master/www/ckd_workflow.png" width="100%", align="middle">.